Form-Controls/index.html

```diff
@@ -1,27 +1,79 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title>My form exercise</title>
-    <meta name="description" content="" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-  </head>
-  <body>
-    <header>
-      <h1>Product Pick</h1>
-    </header>
-    <main>
-      <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
-      </form>
-    </main>
-    <footer>
-      <!-- change to your name-->
-      <h2>By HOMEWORK SOLUTION</h2>
-    </footer>
-  </body>
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>T-Shirt Order Form</title>
+  <link rel="stylesheet" href="style.css" />
+</head>
+<body>
+  <form aria-label="T-Shirt Order Form">
+    <div>
+      <label for="name">Name</label>
+      <input type="text" name="name" id="name" required />
+    </div>
+
+    <div>
+      <label for="password">Password</label>
```

Comment:
Is the password field required in the exercise? 

💭 As a developer, we are sometimes inclined to want to include 'extras' in our work. I suggest getting in the practice of **_not_** doing this as it can introduce extra functionality that is often not required (look up 'scope creep').